### ISSUE TITLE
[issue]: Fallback Fonts feature not working for CJK .ass files

### ISSUE THREAD

**Issue by hoonlight** [jellyfin/jellyfin#10693]
### Please describe your bug

Hello,

On Jellyfin 10.8.13 (Docker), I was able to use the Fallback Fonts feature to path a woff2 font file and display CJK .ass subtitles correctly.
This worked well for videos requiring transcoding as well as for Direct Play.

However, when I used Jellyfin Unstable (10.9.0, Docker) with the exact same settings, all CJK characters were displayed as boxes. 
This is true even when transcoding. Fallback Fonts feature is not working.

Strangely, if I check Force burn subtitles - All complex formats in the settings, the subtitles display fine in the fallback font when transcoding, but they still show up as boxes in Direct Play.

I'm testing 10.8.13 and 10.9.0 on Docker right now with the same settings.

### Jellyfin Version

Other

### if other:

10.9.0

### Environment

```markdown
- OS: Ubuntu 23.10
- Linux Kernel:Linux 6.5.0-14-generic x86_64
- Virtualization: Docker
- Clients: MacOS, Android, Windows
- Browser: Chrome, Safari, Edge
- FFmpeg Version: Jellyfin Default
- Playback Method: Direct Play, Transcode
- Hardware Acceleration: Quicksync
- GPU Model: Intel N100 HD Graphics
- Plugins: none
- Reverse Proxy: none
- Base URL: none
- Networking: host
- Storage: local
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hoonlight** [comment]
For reference, I tested with the Korean subtitles below.
[Suzume (2022) - [2160P].ass.zip](https://github.com/jellyfin/jellyfin/files/13608533/Suzume.2022.-.2160P.ass.zip)

**Comment by barronpm** [comment]
Does the issue occur if you run 10.9 web against a 10.8 server?

**Comment by hoonlight** [comment]
> Does the issue occur if you run 10.9 web against a 10.8 server?

No, I've run 10.8 and 10.9 servers independently, in different dockers, connected to different ports, and the problem is only with 10.9. I didn't make any changes to the web version.

**Comment by felix920506** [comment]
Was able to reproduce with unstable, marking as confirmed

**Comment by crobibero** [comment]
Just to confirm my understanding of the issue- direct playing media doesn't use the fallback fonts, but if the subtitles are burned into the video they display correctly?

If so- I think this is a web issue and not a server one.

Also I don't see the subtitle being burned-in in either ffmpeg log

**Comment by felix920506** [comment]
@crobibero Yes, if the video is burned in then the fonts display normally.

It is when subtitles are rendered in web that fallback fonts aren't getting used. 

I just did a quick test with unstable web on 10.8.13 server, subs are also broke where fallback fonts are not being used and the text is showing up as boxes.

Looks like this is indeed a web issue then
